<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ShaderMan</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="TreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
      <widget class="QWidget" name="editorWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>6</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>2</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="TabEditor" name="tabEditorWindow">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="tabsClosable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="showButtonLayout" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="showLogButton">
                 <property name="text">
                  <string>Show Log</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>158</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="logLayout" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="widget_3" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="hideLogButton">
                   <property name="text">
                    <string>Hide Log</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>158</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="logViewer">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="QOGL_Control" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="OGL_layout" name="GL_Window_underlay" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>3</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="OGL_buttons" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1">
            <widget class="QPushButton" name="oglButton0">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="oglButton8">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>8</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="oglButton1">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="oglButton4">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="oglButton7">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="oglButton3">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="oglButton2">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="oglButton6">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="oglButton5">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="widget_6" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="zmButton13">
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Z- (13)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="xmButton9">
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>X- (9)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_8" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="ypButton12">
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Y+ (12)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="ymButton11">
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Y- (11)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_7" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="zpButton14">
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Z+ (14)</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="xpButton10">
                   <property name="minimumSize">
                    <size>
                     <width>30</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>X+ (10)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="action_Save"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionBuild_Run"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="ProfileWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionBuild_Run"/>
   </widget>
   <widget class="QMenu" name="menuVariables">
    <property name="title">
     <string>&amp;Variables</string>
    </property>
    <addaction name="actionSet_Uniform_Variables"/>
    <addaction name="action_Create_Uniform_Variables"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>&amp;Model</string>
    </property>
    <addaction name="action_LoadModel"/>
    <addaction name="action_Attach_model"/>
   </widget>
   <widget class="QMenu" name="menuTextures">
    <property name="title">
     <string>&amp;Texture</string>
    </property>
    <addaction name="action_TextureSettings"/>
   </widget>
   <widget class="QMenu" name="menu_Programs">
    <property name="title">
     <string>&amp;Programs</string>
    </property>
    <addaction name="action_Program_manager"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>V&amp;iew</string>
    </property>
    <addaction name="action_Show_draw_time_statistics"/>
   </widget>
   <widget class="QMenu" name="menuP_rojectSettings">
    <property name="title">
     <string>P&amp;roject Settings</string>
    </property>
    <addaction name="action_OpenGL_settings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBuild"/>
   <addaction name="menuVariables"/>
   <addaction name="menu_Programs"/>
   <addaction name="menuModel"/>
   <addaction name="menuTextures"/>
   <addaction name="menuP_rojectSettings"/>
   <addaction name="menu_View"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
  </action>
  <action name="actionBuild_Run">
   <property name="text">
    <string>Build&amp;&amp;&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New &amp;Project</string>
   </property>
  </action>
  <action name="actionSet_Uniform_Variables">
   <property name="text">
    <string>&amp;Manage Uniform Variables</string>
   </property>
  </action>
  <action name="action_Create_Uniform_Variables">
   <property name="text">
    <string>&amp;Create Uniform Variables</string>
   </property>
  </action>
  <action name="action_LoadModel">
   <property name="text">
    <string>&amp;Load Model</string>
   </property>
  </action>
  <action name="action_Show_values">
   <property name="text">
    <string>&amp;Show values</string>
   </property>
  </action>
  <action name="action_Attach_model">
   <property name="text">
    <string>&amp;Attach Model</string>
   </property>
  </action>
  <action name="action_TextureSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_Program_manager">
   <property name="text">
    <string>&amp;Program manager</string>
   </property>
  </action>
  <action name="action_Show_ameasure_window">
   <property name="text">
    <string>&amp;Show measure window</string>
   </property>
  </action>
  <action name="action_Show_draw_time_statistics">
   <property name="text">
    <string>&amp;Show Draw Statistics</string>
   </property>
  </action>
  <action name="action_OpenGL_settings">
   <property name="text">
    <string>&amp;OpenGL Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OGL_layout</class>
   <extends>QWidget</extends>
   <header>ogl_layout.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabEditor</class>
   <extends>QTabWidget</extends>
   <header>tabeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>treeview.h</header>
  </customwidget>
  <customwidget>
   <class>ProfileWidget</class>
   <extends>QWidget</extends>
   <header location="global">profiling/profilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
